dora/integration/fuse/src/main/java/alluxio/fuse/options/FuseOptions.java

```diff
@@ -31,17 +31,17 @@
 public class FuseOptions {
   private static final Logger LOG = LoggerFactory.getLogger(FuseOptions.class);
   public static final PropertyKey FUSE_UPDATE_CHECK_ENABLED =
-      PropertyKey.Builder.booleanBuilder("fuse.update.check.enabled")
+      PropertyKey.Builder.booleanBuilder("alluxio.fuse.update.check.enabled")
           .setIsBuiltIn(false)
-          .setDefaultValue(false)
+          .setDefaultValue(true)
           .buildUnregistered();
 
   /**
    * The UFS root that Fuse mounts.
    * In standalone Fuse SDK, this is different from {@link PropertyKey#DORA_CLIENT_UFS_ROOT}.
    * */
   public static final PropertyKey FUSE_UFS_ROOT =
-      PropertyKey.Builder.stringBuilder("fuse.ufs.root")
+      PropertyKey.Builder.stringBuilder("alluxio.fuse.ufs.root")
```

Comment:
@dbw9580  will this rename create issue?

Reply:
Renaming this should not cause breakage in functionality, but I purposely did not follow the naming convention to indicate that this a property should be limited to the fuse component, and should not be used elsewhere.